traffic_portal/test/integration/CommonUtils/API.ts

```diff
@@ -296,7 +296,7 @@ export class API {
             const rand = () => Math.floor(Math.random()*255)+1;
             data.ipAddress = `${rand()}.${rand()}.${rand()}.${rand()}`;
         }
-        if(hasProperty(data, 'name')) {
+        if(hasProperty(data, 'name') && !hasProperty(data, 'noRandomize')) {
```

Comment:
Does this need to handle if `noRandomize` is `false`?

Reply:
Making sure `noRandomize` is not `false` in 3eda6cba43